tools/spotless/openremote-license-header.txt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright $YEAR, OpenRemote Inc.
+ *
+ * See the CONTRIBUTORS.txt file in the distribution for a
+ * full listing of individual contributors.
+ *
```

Comment:
The Git history keeps track of all contributors so we could remove the file and reference to it:

```suggestion
```